### PR TITLE
Add REST API for Colibri2

### PR DESCRIPTION
**Pull request by JonathanLennox**

### CODE REVIEW

**Comment by JonathanLennox** [comment]
So far this just has the serializers/deserializers.

**Comment by JonathanLennox** [comment]
I haven't added anything for the GET to the REST URLs, but I don't currently think those are useful; at the very least, I don't know what the use case is for them.

**Comment by JonathanLennox** [comment]
Jenkins, test this please.

File: jvb/src/main/java/org/jitsi/videobridge/rest/root/colibri/v2/conferences/Conferences.java [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+                throw new BadRequestException();
+            }
+        }
+        catch (ParseException pe)
+        {
+            throw new BadRequestExceptionWithMessage(
+                    "Failed to create conference, could not parse JSON: " + pe.getMessage());
+        }
+
+        ConferenceModifyIQ conferenceModifyIQ
```
Comment (bgrozev) [inline jvb/src/main/java/org/jitsi/videobridge/rest/root/colibri/v2/conferences/Conferences.java:102]: Maybe try/catch any deserialization exceptions? They should be equivalent to bad JSON

**Review by bgrozev — APPROVED** [review]

**Pull request merged**